src/components/Navigation.js

```diff
@@ -10,13 +12,23 @@ export default function Navbar() {
   return (
     <header>
       <nav>
-        <ul className="nav-items">
-          {links.map((link) => (
-            <li key={link.text}>
-              <NavLink to={link.path}>{link.text}</NavLink>
-            </li>
-          ))}
-        </ul>
+        <div className="nav-details">
+          <h2>Bookstore CMS</h2>
+          <ul className="nav-items">
+            {links.map((link) => (
+              <li key={link.text}>
+                <NavLink to={link.path}>{link.text}</NavLink>
+              </li>
+            ))}
+          </ul>
+        </div>
+        <div className="nav-right">
+          <div>
+            <a href="/public/index.html" aria-label="user">
+              <FontAwesomeIcon icon={faUser} className="Mask" style={{ color: '#0290ff' }} />
+            </a>
+          </div>
```

Comment:
- [ ] Kindly decrease the space between nav links and move them to the left beside the logo, not in the middle as you did to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/230731414-89ea7a00-837f-44fa-8d89-cf49a2bb1f8f.png) | ![image](https://user-images.githubusercontent.com/68796232/230731408-e01a9b00-307f-474b-b82e-ac4236e6630b.png)  |